packages/fluentui/react-northstar/src/components/SplitButton/SplitButton.tsx

```diff
@@ -22,8 +22,14 @@ import MenuButton, { MenuButtonProps } from '../MenuButton/MenuButton';
 import { MenuProps } from '../Menu/Menu';
 import { MenuItemProps } from '../Menu/MenuItem';
 import { PopupProps } from '../Popup/Popup';
-
-export interface SplitButtonProps extends UIComponentProps, ChildrenComponentProps, ContentComponentProps {
+import { Ref } from '@fluentui/react-component-ref';
+import { PopperShorthandProps } from '../../utils/positioner/types';
+
+export interface SplitButtonProps
+  extends UIComponentProps,
+    ChildrenComponentProps,
+    ContentComponentProps,
+    PopperShorthandProps {
```

Comment:
```suggestion
    PositioningProps {
```

Currently it's the same, but it can be changed. `PopperShorthandProps` is used in cases when we have `Popper` around a slot

Reply:
Done